superset-frontend/packages/superset-ui-demo/storybook/stories/superset-ui-chart/ChartDataProviderStories.tsx

```diff
@@ -35,7 +35,7 @@ import {
   sankeyFormData,
   sunburstFormData,
   wordCloudFormData,
-} from '@superset-ui/core/test/chart/fixtures/formData';
+} from '../../../../superset-ui-core/test/chart/fixtures/formData';
```

Comment:
Consideration: maybe we could retain the aliases in the config to avoid these long relative paths

Reply:
Good question! Let me explain why I use relative paths here.

`@superset-ui/core/test/` is an incorrect import path, because `superset-ui/core` link to `/superset-frontend/packages/superset-ui-core/src`.

I prepared to add a separated PR to adjust importing statement, contains:`@superset-ui/core/src`, `@superset-ui/core`, `@superset-ui/core/test`

Reply:
Oh right! I'm still rusty after the holidays and didn't notice that it was referencing the test dir. Yes, makes perfect sense!